src/apiURL.ts

```diff
@@ -1 +1,9 @@
+/* eslint-disable prefer-template */
 export const BASE_URL = 'https://dokazovi-be.herokuapp.com/api';
+
+export const OAUTH2_REDIRECT_URI = 'http://localhost:3000/oauth2/redirect';
```

Comment:
can you use relative url here?
localhost doesn't work on any other env but local